### PR TITLE
Wentworth

### PR DESCRIPTION
**Pull request by tccanini**

### CODE REVIEW

File: Zoo/src/filesInJava/Geography.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Created by admin-batch-a on 7/27/17.
+ */
+public class Geography {
+
+    public void writeToFile() throws IOException {
+        String filePath = "/home/admin-batch-a/Desktop/TestDevelopment/Projects/modern.core.java.repo/Zoo";
```
Comment (edwinchukus) [inline Zoo/src/filesInJava/Geography.java]: This code will give error. There is no file name of it. Kindly double-check mine, it has like "other_folders/employee.txt" which is missing in your case.

File: Zoo/src/filesInJava/Geography.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        fileWriter.write("Discovery Channel \n");
+        fileWriter.write("National Geographical Channel \n");
+        fileWriter.write("TAnimal Farm \n");
+
+        fileWriter.flush();
+        fileWriter.close();
+    }
+
+    public void readDataFromFile() throws Exception {
+        String filePath = ("/home/admin-batch-a/Desktop/TestDevelopment/Projects/modern.core.java.repo/Zoo");
```
Comment (edwinchukus) [inline Zoo/src/filesInJava/Geography.java]: This code will give error. There is no file name of it. Kindly double-check mine, it has like "other_folders/employee.txt" which is missing in your case.

**Review by edwinchukus — CHANGES_REQUESTED** [review]

File: Zoo/src/filesInJava/Geography.java (hunk 1 above)
Comment (tccanini) [reply Zoo/src/filesInJava/Geography.java]: i have added a .txt file within the filepath line on the existing code

File: Zoo/src/filesInJava/Geography.java (hunk 2 above)
Comment (tccanini) [reply Zoo/src/filesInJava/Geography.java]: All amended and seemed ok now..
please could review and approve.

thank you

**Review by edwinchukus — APPROVED** [review]
Excellent!!!

**Pull request merged**